src/snowflake/cli/api/commands/flags.py

```diff
@@ -303,15 +282,15 @@ def _diag_log_path_callback(path: str):
     "--format",
     help="Specifies the output format.",
     case_sensitive=False,
-    callback=_callback(lambda: get_cli_context_manager().set_output_format),
+    callback=_context_callback("output_format"),
     rich_help_panel=_CLI_BEHAVIOUR,
 )
 
 SilentOption = typer.Option(
     False,
     "--silent",
     help="Turns off intermediate output to console.",
-    callback=_callback(lambda: get_cli_context_manager().set_silent),
+    callback=_context_callback("silent"),
```

Comment:
I'm not sold on this one. The setters allowed us to get rid of string variables. With proposed implementation if you change attribute/value name auto-refactor won't work

Reply:
Hmm, good point. I can re-add the boilerplate per-callback, but considering that it needs to return the set value as well, it's going to be a lot more code per line, something like...

```python
def callback(silent):
  get_cli_context_manager().silent = silent
  return silent

[...]
  callback=callback,
```

Can you think of a way of reducing it further while directly accessing the attrs?